src/routes/index.tsx

```diff
@@ -86,9 +88,20 @@ const RoutesList = () => {
   const routes = PAGES.filter((r) => r.element)
   return (
     <Routes>
-      {routes.map(({ path, element }) => (
-        <Route key={path} path={path} element={element} />
-      ))}
+      {routes.map(({ path, element }) => {
+        const Element = lazy(() => import(element as string))
```

Comment:
can we avoid the `as string` part? Maybe if you'll define `Page` type it will help